packages/app-project/src/components/AuthModal/shared/components/withFormik/withFormik.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Formik } from 'formik'
+
+function withFormik (WrappedComponent) {
+  class FormikHOC extends React.Component {
+    constructor () {
+      super()
+
+      this.handleBlurWithCallback = this.handleBlurWithCallback.bind(this)
+      this.handleChangeWithCallback = this.handleChangeWithCallback.bind(this)
+    }
+
+    handleBlurWithCallback (event, handleBlur, formikProps) {
+      handleBlur(event)
+      this.props.onBlur(event, formikProps)
+    }
+
+    handleChangeWithCallback (event, handleChange, formikProps) {
+      handleChange(event)
+      this.props.onChange(event, formikProps)
```

Comment:
Should `formikProps.handleChange()` be called from this function too?

Reply:
That's what the `handleChange` is calling in the above line. I wanted to make the rest of the props available to the props handler, like setting the error message of necessary. I see I can tidy this up though.